app/livechat/server/lib/Livechat.js

```diff
@@ -819,6 +822,11 @@ export const Livechat = {
 			}),
 		]));
 
+		const { requestTagBeforeClosingChat, chatClosingTags } = departmentData;
+		if (requestTagBeforeClosingChat && (!chatClosingTags || [...chatClosingTags].length === 0)) {
```

Comment:
Why need the spread here? It was checked before and for sure will be an array if it exists

Reply:
You're right. There is a check of that variable.